source/codegen/metadata/nifake/functions.py

```diff
@@ -1176,6 +1176,41 @@
         'returns': 'ViStatus',
         'use_session_lock': False
     },
+    'GetPatternPinIndexes':{
```

Comment:
Can you rename this method, paramaters, and the corresponding test to be more generic since it's in nifake? Something like `GetAnIviDanceWithATwistArray`.

Reply:
yes, I renamed it